docs/INPUT_FILES_EXPLAINED.md

```diff
@@ -0,0 +1,92 @@
+Note that the first `Boolean` `stand_alone` flag is specific to BMI extention and is not present in source code.
+More on `stand_alone` can be found [here](./STAND_ALONE.md).
+
+<p align="center">
+  <img src="https://github.com/madMatchstick/topmodel/blob/doc-update-bmi-v2.0/docs/img/topmod_run.PNG" width=45% height=45% >
+</p>
+
+| Variable | Datatype | Limits | Role | Description |
+| -------- | -------- | ------ | ---- | ----------- |
+| stand_alone | *int* | Boolean | option | `1`(`TRUE`) or `0` (`FALSE`) flag to indicate if model is to be run via *stand alone* mode; added for BMI adaption |
+| title          | *char* | 256 | info_string | character title of model |
+| \*input_fptr   | FILE   | 256 | file_offset | file pointer to `inputs.DAT`  |
+| \*subcat_fptr  | FILE   | 256 | file_offset | file pointer to `subcat.DAT`  |
+| \*params_fptr  | FILE   | 256 | file_offset | file pointer to `params.DAT`  |
+| \*output_fptr  | FILE   | 256 | file_offset | file pointer to `topmod.OUT`  |
+| \*out_hyd_fptr | FILE   | 256 | file_offset | file pointer to `hyd.OUT`     |
+
+Note\*: All file pointers must be a full relative path; e.g. `data/inputs.DAT`.
+
+## Input Data
+See [`inputs.dat`](../data/inputs.dat).
+
+<p align="center">
+  <img src="https://github.com/madMatchstick/topmodel/blob/doc-update-bmi-v2.0/docs/img/inputs_dat.PNG" width=45% height=45% >
+</p>
+
+| Variable | Datatype | Units | Role | Description |
+| -------- | -------- | ----- | ---- | ----------- |
+| nstep  | *int* |   | time_info | total number of simulation periods |
+| dt     | *int* | hours  | time_info | size of timestep  |
+| rain   | *double* |  meters/hour | input_from_file\* | rainfall rate  |
+| pe     | *double* |  meters/hour | input_from_file\* | potential evapotranspiration  |
+| Qobs   | *double* |  meters/hour | input_from_file\* | observed discharge   |
+
+Note\*: Variable role is "input_from_bmi" if not in stand-alone mode.
+
+## Subcatchment Data
+See [`subcat.dat`](../data/subcat.dat).
+
+<p align="center">
+  <img src="https://github.com/madMatchstick/topmodel/blob/doc-update-bmi-v2.0/docs/img/subcat_dat.PNG" width=75% height=75% >
+</p>
+
+| Variable | Datatype | Limits | Units | Role | Process | Description |
+| -------- | -------- | ------ | ----- | ---- | ------- | ----------- |
+| num_sub_catchments | *int* |   |   | array_length |   | number of subcatments; BMI adaption always sets to 1 as loop to be handled by framework  |
+| imap | *int* | Boolean  |   | option |   | ordinarily tells code to write map; NOT IMPLEMENTED |
+| yes_print_output | *int*  | Boolean |   | option |   | set equal to `1` to print output files |
+| subcat   | *char* | 256 |   | info_string |   | the name of each sub-catchment  |
+| num_topodex_values  | *int* |   |   | parameter_fixed | rainfall-runoff | number of topodex histogram values |
+| area  | *double* | % | meters^2 | parameter_fixed |   | catchment area as % to whole catchment (set to 1) |
```

Comment:
If whole catchment is set to 1, is this a decimal percent (or decimal fraction) as opposed to a % (e.g. 50%)?  The user would input 0.5 if decimal fraction (or percent) rather than % which may be confused with 50.

Reply:
good catch.  just adjusted